frontend/src/scenes/error-tracking/SymbolSetUploadModal.tsx

```diff
@@ -7,42 +7,61 @@ import { LemonField } from 'lib/lemon-ui/LemonField'
 import { errorTrackingSymbolSetLogic } from './errorTrackingSymbolSetLogic'
 
 export const SymbolSetUploadModal = (): JSX.Element => {
-    const { setUploadSymbolSetReference } = useActions(errorTrackingSymbolSetLogic)
-    const { uploadSymbolSetReference, isUploadSymbolSetSubmitting, uploadSymbolSet } =
-        useValues(errorTrackingSymbolSetLogic)
+    const { setUploadSymbolSetId } = useActions(errorTrackingSymbolSetLogic)
+    const { uploadSymbolSetId, isUploadSymbolSetSubmitting, uploadSymbolSet } = useValues(errorTrackingSymbolSetLogic)
 
-    const onClose = (): void => setUploadSymbolSetReference(null)
+    const onClose = (): void => setUploadSymbolSetId(null)
 
     return (
-        <LemonModal title="" onClose={onClose} isOpen={!!uploadSymbolSetReference} simple>
+        <LemonModal title="" onClose={onClose} isOpen={!!uploadSymbolSetId} simple>
             <Form logic={errorTrackingSymbolSetLogic} formKey="uploadSymbolSet" className="gap-1" enableFormOnSubmit>
                 <LemonModal.Header>
-                    <h3>Upload source map</h3>
+                    <h3>Upload javscript symbol set</h3>
                 </LemonModal.Header>
                 <LemonModal.Content className="space-y-2">
-                    <LemonField name="files">
+                    <LemonField name="minified">
                         <LemonFileInput
-                            accept="text/plain"
+                            accept="text/*"
                             multiple={false}
                             callToAction={
-                                <div className="flex flex-col items-center justify-center space-y-2 border border-dashed rounded p-4">
+                                <div className="flex flex-col items-center justify-center space-y-2 border border-dashed rounded p-4 w-full">
                                     <span className="flex items-center gap-2 font-semibold">
-                                        <IconUploadFile className="text-2xl" /> Add source map
+                                        <IconUploadFile className="text-2xl" /> Add minified source
                                     </span>
                                     <div>
-                                        Drag and drop your local source map here or click to open the file browser.
+                                        Drag and drop your minified source file here or click to open the file browser.
                                     </div>
                                 </div>
                             }
                         />
                     </LemonField>
+                    <LemonField name="sourcemap">
+                        <LemonFileInput
+                            accept="text/*"
```

Comment:
Let's double check the types of the two files we need and see if it's possible to make this a little more scoped